src/plugins/hotkeys/hotkeys.ts

```diff
@@ -0,0 +1,132 @@
+import type Player from 'core/player.js'
+
+type pluginParameter = {
+	player: Player
+	options: {
+		seekTime?: number
+		volumeStep?: number
+	}
+}
+
+/**
+ * Vlitejs Hotkeys plugin
+ * @module Vlitejs/plugins/Hotkeys
+ */
+export default class Hotkeys {
+	player: any
+	seekTime: number
+	volumeStep: number
+
+	providers = ['html5', 'youtube', 'dailymotion', 'vimeo']
+	types = ['video', 'audio']
+
+	/**
+	 * @constructor
+	 * @param options
+	 * @param options.player Player instance
+	 * @param options.options Plugins options
+	 */
+	constructor({ player, options = {} }: pluginParameter) {
+		this.player = player
+		this.seekTime = options.seekTime ?? 5
+		this.volumeStep = options.volumeStep ?? 0.1
+
+		this.onKeydown = this.onKeydown.bind(this)
+	}
+
+	/**
+	 * Initialize
+	 */
+	init() {
+		this.addEvents()		
+	}
+
+	/**
+	 * Add event listeners
+	 */
+	addEvents() {
+		this.player.container.addEventListener('keydown', this.onKeydown)
+	}
+
+	/**
+	 * On keydown event on the media element
+	 * @param e Event listener datas
+	 */
+	onKeydown(e: KeyboardEvent) {
+		const activeElement = document.activeElement
+		const { keyCode } = e
+
+		if (
+			[9, 32, 37, 39].includes(keyCode) &&
+			this.player.autoHideGranted &&
```

Comment:
`autoHideGranted` is a property from `Vlitejs` class and can be access with `this.player.Vlitejs.autoHideGranted`.

There are example in Player and Cast plugin